### ISSUE TITLE
mpiexec launcher not supported

### ISSUE THREAD

**Issue by dianarg** [geopm/geopm#451]
```
> geopmlaunch impi -n 1 --ppn 1 --geopm-ctl=process --geopm-report=test_report -- geopmbench
<geopmpy.launcher> Unsupported launcher mpiexec requested
```

**Comment by dianarg** [comment]
After adding mpiexec to the launcher factory, I am also seeing affinity errors:
```> geopmlaunch impi -n 2 --ppn 4 --geopm-ctl=process --geopm-report=test_report -- geopmbench
# produces the error:
<geopm> MPI ranks are not affitinized to distinct CPUs
> OMP_NUM_THREADS=4 geopmlaunch impi -n 2 --ppn 4 --geopm-ctl=process --geopm-report=test_report -- geopmbench
# same error```

**Comment by dianarg** [comment]
Fixed at some point, potentially by 4b7d9af or c2c629a
